mne_bids/datasets.py

```diff
@@ -35,11 +35,11 @@ def fetch_faces_data(data_path=None, repo='ds000117', subject_ids=[1]):
         data is stored.
 
     """
-    if not data_path:
+    if data_path is None:
         home = os.path.expanduser('~')
         data_path = os.path.join(home, 'mne_data', 'mne_bids_examples')
-        if not os.path.exists(data_path):
-            os.makedirs(data_path)
+    if not os.path.exists(data_path):
+        os.makedirs(data_path)
```

Comment:
can you remove this from the release commit?

Reply:
yes, done through the rebase :-)